xray/utils/resultwriter.go

```diff
@@ -270,51 +341,78 @@ func convertVulnerabilities(jsonTable formats.SimpleJsonResults, run *sarif.Run,
 func getDirectDependenciesFormatted(directDependencies []formats.ComponentRow) string {
 	var formattedDirectDependencies strings.Builder
 	for _, dependency := range directDependencies {
-		formattedDirectDependencies.WriteString(fmt.Sprintf("`%s:%s`, ", dependency.Name, dependency.Version))
+		formattedDirectDependencies.WriteString(fmt.Sprintf("`%s %s`<br/>", dependency.Name, dependency.Version))
```

Comment:
Let's not ignore the error returned by this function.

Reply:
done